src/main/java/frc/robot/commands/AutoAim.java

```diff
@@ -0,0 +1,46 @@
+package frc.robot.commands;
+
+
+import static frc.robot.RobotContainer.*;
+
+import edu.wpi.first.math.MathUtil;
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.math.kinematics.ChassisSpeeds;
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.ParallelCommandGroup;
+import frc.robot.InputManager;
+import frc.robot.subsystems.LimelightInterface;
+
+public class AutoAim extends Command{
+   
+    public AutoAim(){
+        addRequirements(PIVOT, SWERVE);
+    }
+    PIDController w_pid = new PIDController(0.1, 0, 0);
+    
+    @Override
+    public void initialize() {
+        LIMELIGHT_INTERFACE.LEDsOn();
+        SHOOTER.setSpeakerShooterMotorSpeeds();
+        
+    }
+    @Override
+    public void execute() {
+        PIVOT.setCurrentPosition(LIMELIGHT_INTERFACE.speakerSetPoint() + LIMELIGHT_INTERFACE.offsetMultiplier());
```

Comment:
This calculation could be inside LIMELIGHT_INTERFACE so that all calculations relating limelight are contained inside LIMELIGHT_INTERFACE.

This can be done later.